WinImageBuilder.psm1

```diff
@@ -930,6 +1031,8 @@ function New-WindowsCloudImage {
             $configValues = @{
                 "InstallUpdates"=$InstallUpdates;
                 "PersistDriverInstall"=$PersistDriverInstall;
+                "PurgeUpdates"=$PurgeUpdates;
+                "DisableSwap"=$DisableSwap;
             }
 
             Generate-UnattendXml $UnattendXmlPath $unattedXmlPath $image $ProductKey $AdministratorPassword

```

Comment:
you need to put this parameters by name, otherwise if a param is undefined, it will fail